node/pkg/lifecycle/startup/startup.go

```diff
@@ -349,6 +349,18 @@ func MonitorIPAddressSubnets() {
 	for {
 		<-time.After(pollInterval)
 		log.Debugf("Checking node IP address every %v", pollInterval)
+
+		// Every polling interval, try to get the k8s Node and use the latest K8s node IP to configure.
+		if clientset != nil {
+			curK8sNode, err := clientset.CoreV1().Nodes().Get(ctx, k8sNodeName, metav1.GetOptions{})
```

Comment:
I think we need to handle the `err != nil` case here with an explicit `log.WithError(err).Error` so that we don't just swallow it with no evidence.

Reply:
fix it.